pkg/planner/core/preprocess.go

```diff
@@ -252,6 +252,24 @@ type preprocessor struct {
 	resolveCtx *resolve.Context
 }
 
+func (p *preprocessor) schemaReadOnly(dbName pmodel.CIStr) {
```

Comment:
```suggestion
func (p *preprocessor) checkSchemaReadOnly(dbName pmodel.CIStr) {
```
methods better have a verb as the first word, unless it's a property get/setter